src/javascript/app_2/App/app.jsx

```diff
@@ -17,11 +19,13 @@ const App = ({ root_store }) => (
                 <div id='header'>
                     <Header />
                 </div>
-
-                <AppContents>
-                    <Routes />
-                    <PortfolioDrawer />
-                </AppContents>
+                <ErrorBoundary>
+                    <AppContents>
+                        <Routes />
+                        <DevTools />
```

Comment:
Do we need to disable this on production?

Reply:
Yeah. I'll push another commit for that.